### PR TITLE
Update "get involved" section

### LINKED ISSUE #264: Get Involved outdated?
The page focuses heavily on the LMMS mailing list

> See our GitHub Wiki and our bug tracker to get started. Furthermore, the LMMS mailing list is a great place to introduce yourself.

> Public registration on our Wiki is disabled due to spam issues. Please post to the mailing list if you're interested in helping out with the documentation!

> Posting to the mailing list is the best way to reach out to our developers:

As far as I'm aware, this is no longer the most active/best channel of communication. Is this correct?

As a side note, the page also states

> We are in desperate need of new developers to join our team. LMMS is written in C++ using the Qt Framework, so if you know a thing or two about these, don't hesitate to get yourself dirty with coding!

However, elsewhere I believe we encourage people to contribute even if they aren't familiar with C++ and Qt beforehand. Should this perhaps be something along the lines of

> We are in desperate need of new developers to join our team. LMMS is written in C++ using the Qt Framework, but even if you don't have prior experience with these, don't hesitate to get yourself dirty with coding!

?

### PR DESCRIPTION
**Pull request by Spekular**
Properly closes #264

A few questions before merge:
- Should we still mention the mailing list at all?
- Given the [ongoing situation](https://arstechnica.com/gadgets/2021/05/freenode-irc-has-been-taken-over-by-the-crown-prince-of-korea/) with Freenode, should we remove that link? I know that @probonopd (author of AppImage, for anyone unaware) has moved. (Apologies for the ping, but perhaps you have some advice on migrating?)

### CODE REVIEW

**Comment by PauseForAffliction** [comment]
My only experience with the mailing list usually involved someone who wanted to contribute reaching out before being redirected to the Discord server. I soon unsubscribed from it after multiple instances of that.

As for the IRC, I can't say since I've never used it. Though the situation going on with it that you linked provides a decent point to migrate from it if not drop it in my opinion.

File: templates/get-involved.twig
Comment (PauseForAffliction) [inline templates/get-involved.twig:41]: You might consider omitting this paragraph if the wiki registration spam issues are talking about the wiki that's been shut down. I'm not sure if your current documentation has the same issues, but I'm assuming that this is talking specifically about the MediaWiki.

Comment (Spekular) [reply templates/get-involved.twig:41]: I suppose the spam part isn't relevant, but access to edit the docs is still something people need to request access to. Will edit that line before this merges.

**Comment by probonopd** [comment]
> Given the ongoing situation with Freenode, should we remove that link? I know that @probonopd (author of AppImage, for anyone unaware) has moved. (Apologies for the ping, but perhaps you have some advice on migrating?)

I do indeed, credits go to nurupo:
https://gist.github.com/nurupo/91b0ebc7f85059b57ea7108a25ae6c69

(In #AppImage we did it wrong and as a result freenode seized our channel, we can't post there anymore, and they have forcibly re-opened another channel which is using our project name without our permission.)

**Comment by Spekular** [comment]
> I do indeed, credits go to nurupo:
> https://gist.github.com/nurupo/91b0ebc7f85059b57ea7108a25ae6c69

Much thanks for the info! For now I'll remove the IRC link entirely, and point anyone who wants to revive it to this.

**Review by liushuyu — APPROVED** [review]

**Comment by Spekular** [comment]
I don't have merge rights on this repo, does someone want to merge this?

**Review by liushuyu — APPROVED** [review]

**Comment by liushuyu** [comment]
Manually merged in 4129bafd294fb4393241f4ad7da4320839d325b6. GitHub for some reason didn't count that as a merge.

**Comment by trebmuh** [comment]
Following https://github.com/LMMS/lmms.io/issues/334 , I've read here.

@Spekular @liushuyu : I'm "admin" on the Freenode IRC channel. Unfortunately, I didn't see this discussion here since I was moving to a new place and new job on those last week. Just got internet connexion back a few days ago.

Would you mind me to take control over the libera.chat #lmms channel in order to get it back working properly? I'll then ask people on the freenode #lmms channel to move there.

In my opinion and practice, the IRC channel is a good place for people using LMMS who want some quick answers. When occasionally some people proposing to contribute code come there, I'm redirecting then to the github forge.

**Comment by Spekular** [comment]
> @Spekular @liushuyu : I'm "admin" on the Freenode IRC channel.
> 
> Would you mind me to take control over the libera.chat #lmms channel in order to get it back working properly? I'll then ask people on the freenode #lmms channel to move there.

In my opinion this sounds like a good plan. If the freenode channel was worthy of an lmms.io link while under your care, the same should be true of the liberachat one. I think it'd be good if you follow the guide probonopd linked (to migrate the channel) and then open a PR adding the correct new link back to the site.

I'd like to wait a bit first though so others can chime in in case someone objects for some reason.

**Comment by trebmuh** [comment]
@Spekular @tresf : it's been 10 days now. Are you OK for me to move forward and chat with the libera.chat people?

**Comment by tresf** [comment]
Sorry for the delay and thanks for the information.  Please feel free to move and notify with status message. 👍

If there isn't someone to change the IRC status, ping me, I can do it.  Also please make a few of the regulars admins on the new channel (e.g. me, umcaruje, specular) just in case.  Taking over IRC when we lost control took months so distributing access is important. 🍻

**Pull request closed without merging**
